### ISSUE TITLE
CPRuleEditor does not resize the rhs view after the ruleeditor has been shrunken.

### ISSUE THREAD

**Issue by kerusan** [cappuccino/cappuccino#1985]
When resizing a CPRuleEditor the right hand side view (the value control view) does not expand again after being shrunken.

The issue can be demonstrated with the CPRuleEditorCibTest. When displaying the ruleeditor the left hand side view (like a textfield) has a size.

When makeing the window width smaller the textfield shrinks but when widening the window again  the textfield will keep its minimal size.

![screen shot 2013-09-10 at 13 20 29](https://f.cloud.github.com/assets/783541/1113986/93552d8e-1a0c-11e3-9a33-5c929a77494e.png)

![screen shot 2013-09-10 at 13 21 40](https://f.cloud.github.com/assets/783541/1113990/a04cc02e-1a0c-11e3-89ec-044053010b1a.png)

![screen shot 2013-09-10 at 13 21 52](https://f.cloud.github.com/assets/783541/1113991/a3863c66-1a0c-11e3-869c-755f6eeedd6a.png)

**Comment by daboe01** [comment]
@kerusan is this still an issue in the latest master?

**Comment by kerusan** [comment]
Yes the issue still exist. We are now running a “master” of Cappuccino that is a month old, 
but as far as I have seen it is still there. I have not seen any one push or some pullrequest in
this issue since we merged in Cappuccino in Januari.

Our client have us heavily occupied so I cant look into it right now.


> @kerusan <https://github.com/kerusan> is this still an issue in the latest master?

—kerusan

**Comment by daboe01** [comment]
@cacaodev, could this be fixed by adding a ```objc [[ruleView setAutoresizingMask:CPViewWidthSizable]``` to e.g. line 317 in _CPRuleEditorViewSlice.j?
not tried it, though, just guessing...
